lib/utilities/collectResponseHeaders.js

```diff
@@ -15,8 +15,7 @@ function collectResponseHeaders(objectMD, corsHeaders, versioningCfg,
   returnTagCount) {
     // Add user meta headers from objectMD
     const responseMetaHeaders = Object.assign({}, corsHeaders);
-    Object.keys(objectMD).filter(val => (val.substr(0, 11) === 'x-amz-meta-' ||
-        val === 'x-amz-website-redirect-location'))
+    Object.keys(objectMD).filter(val => (val.substr(0, 11) === 'x-amz-meta-'))
```

Comment:
Nothing wrong here, just another method you could use here is `String#startsWith` method and you won't need to specify the length